content/node-runners/testnet2-migration.md

```diff
@@ -1,70 +1,100 @@
+```bash
+ssh myst@ip-of-your-raspberry
+```
 
-We also created user-friendly consumer CLI commands.
-Now it’s much easier to use [Mysterium VPN for advanced Linux users](/user-guide/) and on servers.
+The default password is  `mystberry`.
 
-### Matic x Mysterium
+Then to update your RPI, run:
 
-We started working on our [integration with Matic Network](https://mysterium.network/blog/matic-network-powers-mysterium-p2p-payments/). Most of our infrastructure components are ready for Matic, so now we’re working on building the Matic<>Ethereum bridges needed before it’s public release.
+```bash
+sudo apt update; sudo apt install myst
+```
 
-### My.Mysterium.Network (MMN) updates
+### How to update your Linux node
 
-For a short period of time, the Testnet 2.0 MMN will be hosted on https://testnet2.mysterium.network.
-Once we migrate the majority of nodes, the old https://my.mysterium.network will be replaced with the Testnet 2.0 version.
+To update your Linux node, run:
+```bash
+sudo apt update; sudo apt install myst
+```
 
-**On 2020-01-08** - All of the accounts from https://my.mysterium.network are migrated to https://testnet2.mysterium.network.
+### How to update your Docker node
 
-The Testnet 2.0 version of MMN will look a little different than the current version, and we hope it will solve a couple of usability problems we had before.
+In order to update your Docker node, run the following commands.
 
-Nodes can also store beneficiary (payout) wallet addresses on the Ethereum blockchain, instead of a database. The new MMN has been adapted so it can be used for bounty payments.
+Check the Repository, Tag, Image ID, Creation date and size of the current build.
 
-We also discovered some node runners who were cheating the system. We therefore started a blacklist so they could not participate in any future bounty program.
+```bash
+docker images
+```
 
-### Earn in 2 networks during the migration
+Delete the image that is already being used by your Docker container:
 
-We’ve updated and deployed multiple versions of MMN, so users of Testnet 1.0 & Testnet 2.0 networks could get access to an aggregated node dashboards. 
-VPN services rendered on Testnet 2.0 will run alongside VPN services on Testnet 1.0 in January 2021. **Earn double while we migrate nodes across to Testnet 2.0!**
+```bash
+docker rmi -f ImageID
+```
 
-## Ready to upgrade? Here's how.
+Use the latest docker image:
 
-Firstly you will need to [update your nodes](/node-runners/setup/updating/) and [log into the Node UI](/node-runners/node-ui/).
+```bash
+docker pull mysteriumnetwork/myst:latest
+```
 
-**How to update your current MMN/Node UI settings to be compatible with Testnet 2.0?**
+## Confirm your node is running
 
-<img src="../images/node-ui/welcome.png" alt="Welcome!" class="screenshot">
-<br /><br />
+To confirm that your node is running on Linux and RPi, check the service status:
+```bash
+sudo systemctl status mysterium-node.service 
+```
 
```

Comment:
To confirm that your node is running on a Docker container, enter the following into a terminal window: 
`docker ps `

The ps command provides several columns of information:

Container ID – a unique alphanumeric number for each container
Image – The base operating system image the container is based on
Command – The command that launched the container
Created – How long ago the container was created
Status – Uptime or downtime
Ports – Specifies any ports forwarded to the container for networking
Name – A memorable name assigned by the Docker software